src/style.css

```diff
@@ -1,3 +1,80 @@
 body {
   margin: 0;
+
+  --link-color: #00acdc;
+  --header-height: 60px;
+
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. You should specify a generic family at the end of the font property value. In this case, you can use 'serif' or 'sans-serif'.